FileSystem/FileIOUtil.js

```diff
@@ -212,3 +212,162 @@ function getLineStart(file) {
   }
   return filePointer + 1;
 }
+
+/**
+ * Helper function to parse one variable
+ * @param(object) file File structure
+ * @param {VbaBox} variable reference variable to store value
+ */
+function inputFileUtil(file, variable) {
+  var content = file.content;
+  var filePointer = file.pointer;
+  // Jump over whitespace
+  while (filePointer < content.length && content[filePointer] == ' ') {
+    filePointer++;
+  }
+  // No input left
+  if (filePointer == content.length) {
+    throw new Error('End of file reached');
+  }
+  file.pointer = filePointer;
+  // Try obtaining an input
+  if (!(tryInputConstant(file, variable) || tryInputVariable(file, variable))) {
+    throw new Error('Unknown Expression');
+  }
+  filePointer = file.pointer;
+  // Jump over whitespace and break at any delimiting characters
+  while (filePointer < content.length) {
+    if (content[filePointer] == ' ') {
+      // Jump over whitespace
+      filePointer++;
+    } else if (content[filePointer] == ',') {
+      // Break at delimiter
+      filePointer++;
+      break;
+    } else if (content.substr(filePointer, 2) == '\r\n') {
+      // Break at delimiter
+      filePointer += 2;
+      break;
+    } else if (content[filePointer] == '\r') {
+      // Break at delimiter
+      filePointer++;
+      break;
+    } else if (content[filePointer] == '\n') {
+      // Break at delimiter
+      filePointer++;
+      break;
+    } else {
+      throw new Error('Unknown Error Occurred');
+    }
+  }
+  file.pointer = filePointer;
+}
+
+/**
+ * Helper function to parse constant
+ * @param(object) file File structure
+ * @param {VbaBox} variable reference variable to store value
+ * @return {boolean} true if constant is found
+ */
+function tryInputConstant(file, variable) {
+  // Extract next 7 characters only
```

Comment:
What do you mean by this?

Reply:
Changed the logic slightly, should be clear now.